pkmodel/solution.py

```diff
@@ -7,31 +7,45 @@
 
 
 class Solution:
-    """A Pharmokinetic (PK) model solution
-
-    Parameters
-    ----------
-
-    value: numeric, optional
-        an example paramter
-
-    """
     # def __init__(self, models, protocols, therapeutic_min, therapeutic_max):
     def __init__(self, models, therapeutic_min, therapeutic_max):
+    """Defining the system of differential equations and then 
```

Comment:
@Ramanspec these should all be indented to match the indentation of the code within the function (that's why it's failing the linting)